src/main/java/seedu/jarvis/logic/parser/planner/AddTaskCommandParser.java

```diff
@@ -53,11 +53,15 @@ public AddTaskCommand parse(String args) throws ParseException {
         String taskType = argMultimap.getValue(PREFIX_TASK_TYPE).get();
         String taskDes = argMultimap.getValue(PREFIX_TASK_DES).get();
         Set<Tag> tagList = ParserUtil.parseTags(argMultimap.getAllValues(PREFIX_TAG));
-        System.out.println("hello there");
         LocalDate[] dates = new LocalDate[2];
         if (isEventOrDeadline(argMultimap)) {
-            System.out.println("hi there");
             dates = ParserUtil.parseDate(argMultimap.getValue(PREFIX_DATE).get());
+            if (taskType.equals("event")) {
```

Comment:
This has quite a few magic strings here. I recommend using string constants for these.